versions/models.py

```diff
@@ -557,8 +584,8 @@ def create_versioned_many_to_many_intermediary_model(field, cls, field_name):
         return type(str(name), (Versionable,), {
             'Meta': meta,
             '__module__': cls.__module__,
-            from_: VersionedForeignKey(cls, related_name='%s+' % name),
-            to_field_name: VersionedForeignKey(to, related_name='%s+' % name),
+            from_: VersionedForeignKey(cls, related_name='%s+' % name, auto_created=name),
```

Comment:
I'm not entirely sure whether it's ABusing the auto_created property (which implies, that I'm not super familiar with the ``auto_created`` property). 
Resources that I've found are:
- https://docs.djangoproject.com/en/1.7/howto/custom-model-fields/#writing-a-field-subclass (which says: *auto_created: True if the field was automatically created, as for the OneToOneField used by model inheritance. For advanced use only.*)
- http://django-model-internals-reference.readthedocs.org/en/latest/auto_created.html (which would probably be a good resource, but which is still tagged as a TODO and empty otherwise)

Depending on what 'created' means, our use case may not be that wrong. 
Also considering Django sets the ``auto_created`` flag on [intermediary models in M2M relationships](https://github.com/django/django/blob/stable/1.7.x/django/db/models/fields/related.py#L1847), why not setting it on its ForeignKeys? (It's true that I'm a bit worried that Django doesn't set that flag on ForeignKeys being part of auto_created intermediary models.) 

However, side effects would need to be stress tested. Migrations are a good keyword.